contexts/server-base/Dockerfile

```diff
@@ -71,7 +71,7 @@ ARG TARGETARCH
 ARG UBUNTU_VERSION
 
 # curl is necessary for some kafka integration testing
-# unixodbc, odbc-postgresql for postgres  / turbodbc integration testing
+# unixodbc, odbc-postgresql for postgres integration testing
```

Comment:
Strictly we don't need these packages, but we want to keep them in case we need to do manual testing of turbodbc.